docs/experiments/overview.md

```diff
@@ -117,3 +97,33 @@ For example:
 ## Generic pattern matching
 
 See [generic pattern matching](../generic-pattern-matching/).
+
+## Deprecated experiments
+
+### Equivalences
+
+:::note
+This feature was deprecated in Semgrep v0.61.0.
+:::
+
+Equivalences enable defining equivalent code patterns (i.e. a commutative property: `$X + $Y <==> $Y + $X`). Equivalence rules use the `equivalences` top-level key and one `equivalence` key for each equivalence.
+
+For example:
+
+<iframe src="https://semgrep.dev/embed/editor?snippet=jNnn" border="0" frameBorder="0" width="100%" height="435"></iframe>
+
+### Taint tracking
+
+:::note
+This feature was deprecated in Semgrep v0.61.0.
```

Comment:
Same question as above, is there an alternative to now use?

Reply:
Our newest taint improvements but they’re not yet documented.

Reply:
For the PA team taint-tracking not only is not a deprecated feature but, in fact, it is a priority to push taint-tracking from experimental to beta. Am I parsing this PR the wrong way?